### PR TITLE
feat(FEC-13187): transcript plugin layout on mobile breakpoint

### PR DESCRIPTION
**Pull request by semarche-kaltura**
solves: https://kaltura.atlassian.net/browse/FEC-13187

Screenshot 2023-07-13 at 16 31 59
Screenshot 2023-07-13 at 16 33 56

### CODE REVIEW

File: src/components/caption/caption.tsx [hunk 1]
```diff
@@ -113,42 +98,34 @@ export class Caption extends Component<ExtendedCaptionProps, CaptionState> {
...
-    this.setState({
-      focused: false
-    });
-  };
-
   render() {
     const {caption, highlighted, showTime, longerThanHour} = this.props;
     const {startTime, id} = caption;
     const isHighlighted = Object.keys(highlighted)[0] === id;
+    const time = showTime ? secontsToTime(startTime, longerThanHour) : '';
```
Comment (SivanA-Kaltura) [inline src/components/caption/caption.tsx:105]: seconds

File: src/components/caption/caption.tsx [hunk 2]
```diff
@@ -113,42 +98,34 @@ export class Caption extends Component<ExtendedCaptionProps, CaptionState> {
...
+      <A11yWrapper onClick={this._handleClick}>
         <div
           className={styles.caption}
           ref={node => {
             this._hotspotRef = node;
           }}
           {...captionA11yProps}>
-          {showTime && <div className={styles.captionTime}>{secontsToTime(startTime, longerThanHour)}</div>}
+          {showTime && (
+            <div className={styles.captionTime} aria-hidden="true">
```
Comment (SivanA-Kaltura) [inline src/components/caption/caption.tsx:123]: why aria-hidden ? this element shouldn't be exposed ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

**Comment by SivanA-Kaltura** [comment]
@semarche-kaltura note that for change detection and for conventional commits, "feat" in the commit name should be in lowercase

File: src/components/caption/caption.tsx (hunk 2 above)
Comment (semarche-kaltura) [reply src/components/caption/caption.tsx:123]: yes, it shouldn't be exposed. The parent node (button that user can click) has aria label with the data and screen-reader reads the captionTime and captionContent.
Current approach I took from youtube transcript feature.

**Pull request merged**